pkg/config/config.go

```diff
@@ -24,6 +24,9 @@ const (
 	// which specifies the Namespace to watch.
 	// An empty value means the operator is running with cluster scope.
 	WatchNamespaceEnvVar = "WATCH_NAMESPACE"
+	// ValidationModeEnvVar is the constant for env variable VALIDATION_MODE
+	// It allows to override default validationMode setting for ExtendedDaemonSetSpecStrategyCanary.
+	ValidationModeEnvVar = "VALIDATION_MODE"
```

Comment:
Can we add a prefix to the env var, something like `EDS_VALIDATION_MODE`
```suggestion
	ValidationModeEnvVar = "EDS_VALIDATION_MODE"
```
we dont have it for the `WATCH_NAMESPACE` because it is an inheritance from the `operator-sdk`